theme/src/js/components/DocSideNav.js

```diff
@@ -44,61 +37,31 @@ class ArticlesList extends preact.Component {
 	getArticles() {
 		const {id, locale} = this.props;
 
-		this.setState(
-			{
-				loading: true
-			}
-		);
+		// 300 is an artibuary large number to get all articles in one swoop
```

Comment:
Should we make this a `const` at the top of the file so that we can update this easily without having to dig through the code later on to find it?

You can add the comment above the `const` so you know what it is used for.

Reply:
Yup, fde0a9f!

Reply:
Thanks @rdai10!